### PR TITLE
Count on table pagination | Features/psp 2691

### PR DESCRIPTION
**Pull request by FuriousLlama**
image

### CODE REVIEW

File: frontend/src/features/leases/list/LeaseSearchResults/LeaseSearchResults.tsx [hunk 1]
```diff
@@ -159,6 +159,8 @@ export function LeaseSearchResults(props: ILeaseSearchResultsProps) {
       onRequestData={updateCurrentPage}
       onPageSizeChange={setPageSize}
       noRowsMessage="Lease / License details do not exist in PIMS inventory"
+      totalItems={props.totalItems}
+      tableToolbarText="This is my text!"
```
Comment (devinleighsmith) [inline frontend/src/features/leases/list/LeaseSearchResults/LeaseSearchResults.tsx:163]: Feel like this was left in by accident.

**Comment by devinleighsmith** [comment]
This doesn't seem right:
image

**Comment by devinleighsmith** [comment]
The story specifies this:
At the moment pagination can be seen in, all of these should be reflect this update:
Users list
Lease list
Properties list
Contacts list 

But currently I only see numbers on the Lease list.

File: frontend/src/features/leases/list/LeaseSearchResults/LeaseSearchResults.tsx (hunk 1 above)
Comment (FuriousLlama) [reply frontend/src/features/leases/list/LeaseSearchResults/LeaseSearchResults.tsx:163]: good one!

**Comment by devinleighsmith** [comment]
Thanks, approved.

**Review by devinleighsmith — APPROVED** [review]

**Pull request merged**